src/main/java/fi/hsl/common/hfp/HfpJson.java

```diff
@@ -12,8 +12,8 @@
     //Example payload:
     // {"VP":{"desi":"81","dir":"2","oper":22,"veh":792,"tst":"2018-04-05T17:38:36Z","tsi":1522949916,"spd":0.16,"hdg":225,"lat":60.194481,"long":25.03095,"acc":0,"dl":-25,"odo":2819,"drst":0,"oday":"2018-04-05","jrn":636,"line":112,"start":"20:25"}}
 
-    @JsonAttribute(nullable = false)
-    public Payload VP;
+    @JsonAttribute(nullable = false, name = "VP", alternativeNames = {"DUE", "ARR", "DEP", "ARS", "PDE", "PAS", "WAIT", "DOO", "DOC", "TLR", "TLA", "DA", "DOUT", "BA", "BOUT", "VJA", "VJOUT"})
```

Comment:
So you would call deserialized object like `json.payload` instead of `json.VP`?

Reply:
Yes